1.9/administration/upgrading.md

```diff
@@ -18,153 +18,75 @@ This document provides instructions for upgrading a DC/OS cluster from version 1
 - You must be using systemd 218 or newer to maintain task state.
 - All hosts (masters and agents) must be able to communicate with all other hosts on all ports, for both TCP and UDP.
 - In CentOS or RedHat, install IP sets with this command (used in some IP detect scripts): `$ sudo yum install -y ipset`
 - You must be familiar with using `systemctl` and `journalctl` command line tools to review and monitor service status. Troubleshooting notes can be found at the end of this [document](#troubleshooting).
 - You must be familiar with the [Advanced DC/OS Installation Guide][advanced-install].
 - You should take a snapshot of ZooKeeper prior to upgrading. Marathon supports rollbacks, but does not support downgrades.
+- The full DC/OS version string that you are upgrading from.
+  - In 1.8 this can be found in the lower left corner of the DC/OS UI when screen is maximized.
+  - In 1.9 this can be found under the Cluster menu.
+
+## Supported Upgrade Paths
+
+- 1.8.latest => 1.9.latest
+- 1.9.x => 1.9.(x+1)
+- 1.9.x => 1.9.x (for configuration changes)
 
 ## Instructions
 
 ### Bootstrap Nodes
 
-1.  Copy and update the DC/OS 1.7 `config.yaml` and `ip-detect` files to a new, clean folder on your bootstrap node.
+1.  Copy and update the DC/OS 1.8 `config.yaml` and `ip-detect` files to a new, clean folder on your bootstrap node.
 
     **Important:**
 
     *  You cannot change the `exhibitor_zk_backend` setting during an upgrade.
-    *  The syntax of the DC/OS 1.8 `config.yaml` differs from that of DC/OS 1.7. For a detailed description of the 1.8 `config.yaml` syntax and parameters, see the [documentation](/docs/1.9/administration/installing/custom/configuration-parameters/).
+    *  The syntax of the DC/OS 1.9 `config.yaml` differs from that of DC/OS 1.8. For a detailed description of the 1.9 `config.yaml` syntax and parameters, see the [documentation](/docs/1.9/administration/installing/custom/configuration-parameters/).
 
-1.  After you have merged your 1.7 `config.yaml` into the 1.8 `config.yaml` format, you can build your installer package:
+1.  After you have converted your 1.8 `config.yaml` into the 1.9 `config.yaml` format, you can build your installer package:
 
     1.  Download the file `dcos_generate_config.sh`.
     1.  Generate the installation files.
-    1.  Disable Docker restarts in `dcos_install.sh` by using this command:
-
         ```
-        sed -i -e "s/systemctl restart systemd-journald//g" -e "s/systemctl restart docker//g" genconf/serve/dcos_install.sh
+        dcos_generate_config.sh --generate-node-upgrade-script <installed_cluster_version_upgrading_from>
         ```
-
-        **Important:** This step is critical to prevent task restarts.
-
     1.  Run the [NGINX][advanced-install] container to serve the installation files.
+    1.  A URL will be returned after successfully running the command above. Record this URL for use in later steps. It will be referred to in this document as the "Node upgrade script URL".
```

Comment:
We can make this more specific and say that the URL will be on the last line of stdout and be prefixed with `Node upgrade script URL:`. Here's the test which guarantees that: https://github.com/dcos/dcos/blob/master/dcos_installer/test_backend.py#L91